_pages/StanislavKogan.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: StanislavKogan
+pageUrl: StanislavKogan.github.io
+timestamp: TODAYS-DATE-2017-05-31
```

Comment:
@StanislavKogan  Make sure this format is `timestamp: 2017-05-31`, _not_ `TODAYS-DATE-2017-05-31`